AEXS/aex-141.md

```diff
@@ -405,6 +415,108 @@ stateful entrypoint burn(token_id: int) : unit
 | :--- | :--- |
 | token_id | int |
 
+## Extension Mintable Templates ("mintable_templates")
+
+The `mintable_templates` extension SHOULD be used if multiple NFTs within a contract should share the same metadata. The maximum edition size of NFTs for a specific template needs to be defined on template creation.
+
+```sophia
+contract interface IAEX141MintableTemplates =
+
+    record template =
+        { immutable_metadata_url: string
```

Comment:
So preserving the `metadata_type` and it's use, could we define the field as `immutable_metadata`? If `metadata_type` is an `URL` the user expects on `metadata(token_id)` return a some url but if it's  a `MAP` a rendered of `MetadataMap`. It would help to have a single rule/instruction for both regular and template NFTS.

Reply:
let's postpone this metadata discussion and get the proposal merged to be viewed and commented by a larger audience.

I will provide an update shortly